nimbus.fml.yaml

```diff
@@ -508,6 +508,22 @@ features:
             "move-actions": true,
             "toolbar-changes": true
         }
+  creditcard-autofill:
+    description: This property defines whether credit card autofill is enabled / disabled
+    variables:
+      creditcard-changes:
```

Comment:
Would this be better named as `status`, as it's the status of the feature? `creditcard-changes` isn`t very clear about what exactly it is, and we've used `status` in a few other features as the enabled/disabled key. I should probably document this somewhere, really.

Reply:
good call